### PR TITLE
Change to flake8

### PR DESCRIPTION
**Pull request by hydai**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -12,9 +12,27 @@ python manage.py crawl_course
...
-For auto decaptcha feature, it requires ``tesseract 3.03``
-
 To clear all contents in db, use ``python manage.py crawl_course clear``
 
 To update all syllabus, use ``python manage.py update_syllabus``
 
+For auto decaptcha feature, it requires ``tesseract 3.03``
+
+# Install dependency
+- tesseract
```
Comment (henryyang42) [inline README.md:22]: Just pass Windows!

File: README.md [hunk 2]
```diff
@@ -12,9 +12,27 @@ python manage.py crawl_course
...
+    - mac: `brew install tesseract --devel # Without --devel, it will install tesseract 3.02`
+    - ubuntu: `sudo apt-get install tesseract-ocr`
+
+
+# Setting
+We use mysql as database. The default setting will find a config file in 'NTHU_Course/mysql.ini'. The following is a sample config file:
```
Comment (henryyang42) [inline README.md:28]: Can you create a script which will prompt user to type in these info if the file does not exist? Thanks!

Comment (henryyang42) [inline README.md:28]: Maybe we can combine it with config

File: table/static/js/saveAsFile.js [hunk 3]
```diff
@@ -1,4 +1,10 @@
 $("#saveAsFile").click(function() {
+  var tmp_attr = $("#course-table").attr("class");
+  $("#course-table").removeAttr("class");
+  $("#course-table").css("width", "200%");
```
Comment (henryyang42) [inline table/static/js/saveAsFile.js]: Can you use css style toggle to eliminate these code?

File: README.md (hunk 1 above)
Comment (hydai) [inline README.md:22]: Do we need to support windows?

File: table/static/js/saveAsFile.js (hunk 3 above)
Comment (hydai) [inline table/static/js/saveAsFile.js]: Sure. In progress.

File: README.md (hunk 1 above)
Comment (henryyang42) [inline README.md:22]: Windows users will find their way out.

File: table/static/js/saveAsFile.js (hunk 3 above)
Comment (hydai) [inline table/static/js/saveAsFile.js]: Well, it won't be modified in this PR.
I'll find another way to create this same effect.

**Comment by hydai** [comment]
This PR needs to split to multiple PR.

**Pull request closed without merging**